caas/kubernetes/provider/k8stypes_test.go

```diff
@@ -109,31 +109,36 @@ initContainers:
 service:
   annotations:
     foo: bar
-customResourceDefinition:
-  - group: kubeflow.org
+customResourceDefinitions:
+  mycrd:
```

Comment:
it would be good to use a real crd naming format like:
`tfjobs.kubeflow.org`

Reply:
ok